frontend/services/entities/autofill.ts

```diff
@@ -0,0 +1,11 @@
+// import sendRequest from "services";
+import { sendRequest } from "services/mock_service/service/service";
+import endpoints from "utilities/endpoints";
+
+export default {
+  getPolicyInterpretationFromSQL: (sql: string): Promise<any> => {
```

Comment:
@ghernandez345 ok, I took what you said and I think this is a good fix.

I changed this from `/entities/ai_autofill.ts` to `entities/autofill.ts`
and `getHumanInterpretationFromSQL` to `getPolicyInterpretationFromSQL`

My hesitation on putting in into a policies entity file is that it doesn't really call the policies global or team APIs, but it goes to  `AUTOFILL_POLICIES: `/${API_VERSION}/fleet/autofill/policies`,`

The plan is to add more AI autofills and sorts across our UI and I think if we do more autofills it'll be something like `autofill/query` `autofill/label`

Reply:
Now that I think about it, it should be AUTOFILL_POLICY singular and `fleet/autofill/policy` singular...